### PR TITLE
Remove src/utilities/theme/* and color system docs

### PR DESCRIPTION
**Pull request by alex-page**
- [x] Remove color system docs md file - this will be replaced with a page on polaris-styleguide website
- [x] Remove src/util/theme - we can start using the designTokens directory

### CODE REVIEW

File: src/components/Collapsible/tests/Collapsible.test.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ describe('<Collapsible />', () => {
 
   describe('Transition', () => {
     it('passes a duration property', () => {
-      const duration = Tokens.duration150;
```
Comment (alex-page) [inline src/components/Collapsible/tests/Collapsible.test.tsx]: This is temporary. Unsure why it's using 150ms.

**Review by kyledurand — APPROVED** [review]

File: src/components/Collapsible/tests/Collapsible.test.tsx (hunk 1 above)
Comment (BPScott) [reply src/components/Collapsible/tests/Collapsible.test.tsx]: If this is a boring old test then do we even care about the number coming from a token?

Could we just do `const duration = '100';` and not bother with any imports from `designTokens/motion` at all. Same for the `timingFunction` down below.

That'd also mean that this test update can be done against the main branch asap, to reduce the number of changes in the v8 branch

File: src/components/Collapsible/tests/Collapsible.test.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, {useState, useCallback} from 'react';
 import {mountWithApp} from 'tests/utilities';
 
-import {Tokens} from '../../../utilities/theme';
+import {motion} from '../../../designTokens/motion';
```
Comment (aaronccasanova) [inline src/components/Collapsible/tests/Collapsible.test.tsx]:
```suggestion
import {motion} from '../../../designTokens';
```

**Review by aaronccasanova — APPROVED** [review]
LGTM!

🔥 image 🔥

**Pull request merged**